### PR TITLE
Various plot fixes in ArchR and Seurat task

### LINKED ISSUE #68: Remove dots from violin plot

### LINKED ISSUE #69: bigger labels on seurat plots

### LINKED ISSUE #71: Fix the ATAC log plot
The striping!!!

### PR DESCRIPTION
**Pull request by sidwekhande**
ArchR Density plot fixed, and labels and overall size of Seurat plots fixed. 
MT-genes for mm10 not being found fixed.

Closes #68, closes #69 , closes #71 

Note - the commits in Jan have already been merged into the main, but have reappeared here for unknown reasons

### CODE REVIEW

**Review by emattei — COMMENTED** [review]
Can you also remove the comments and just delete the lines?

File: dockerfiles/share_task_archr.dockerfile [hunk 1]
```diff
@@ -59,9 +59,9 @@ RUN R --no-echo --no-restore --no-save -e "devtools::install_github('immunogenom
 
 RUN R --no-echo --no-restore --no-save -e "remotes::install_version('Seurat', version = '4.1.1')"
 
-RUN R --no-echo --no-restore --no-save -e "install.packages(c('logr','hexbin'))"
+RUN R --no-echo --no-restore --no-save -e "install.packages(c('logr','hexbin', 'ggpointdensity'))"
 
-RUN python3 -m pip install jupyter papermill
+RUN python3 -m pip install --break-system-packages jupyter papermill
```
[the remark this answers is not in the document]
Comment (sidwekhande) [reply dockerfiles/share_task_archr.dockerfile:64]: Python 3.11.2 is now installed,and it recommends installing via venv (base is considered externally managed environment). To bypass and force install,  use the --break-system-packages parameter.

[the remark this answers is not in the document]
Comment (sidwekhande) [reply dockerfiles/share_task_archr.dockerfile:64]: Also, by comments do you mean in the ArchR notebook?

[the remark this answers is not in the document]
Comment (sidwekhande) [reply dockerfiles/share_task_archr.dockerfile:64]: comments removed

**Review by emattei — APPROVED** [review]

**Pull request merged**